src/views/language/LanguageVitalityCell.tsx

```diff
@@ -0,0 +1,99 @@
+import { WifiIcon, WifiHighIcon, WifiLow, WifiOff } from 'lucide-react';
+import React from 'react';
+
+import Hoverable from '../../generic/Hoverable';
+import { LanguageData } from '../../types/LanguageTypes';
+
+import {
+  computeVitalityMetascore,
+  getAllVitalityScores,
+  getVitalityExplanation,
+  VitalityMeterType,
+} from './LanguageVitalityComputation';
+
+export interface LanguageVitalityCellProps {
+  lang: LanguageData;
+  type: VitalityMeterType;
+}
+
+export enum VitalityBucket {
+  Strong = 'Strong',
+  Medium = 'Medium',
+  Low = 'Low',
+  Extinct = 'Extinct',
+  Unknown = 'Unknown',
+}
+
+function getScoreBucket(score: number | null): VitalityBucket {
+  if (score === null) return VitalityBucket.Unknown;
+  if (score >= 7) return VitalityBucket.Strong;
+  if (score >= 4) return VitalityBucket.Medium;
+  if (score >= 1) return VitalityBucket.Low;
+  return VitalityBucket.Extinct;
+}
+
+const BucketIcon: React.FC<{ bucket: VitalityBucket }> = ({ bucket }) => {
+  switch (bucket) {
+    case VitalityBucket.Strong:
+      return <WifiIcon size="1em" />;
+    case VitalityBucket.Medium:
+      return <WifiHighIcon size="1em" />;
+    case VitalityBucket.Low:
+      return <WifiLow size="1em" />;
+    case VitalityBucket.Extinct:
+      return <WifiOff size="1em" />;
```

Comment:
Use Icon namespace here too
```suggestion
      return <WifiLowIcon size="1em" />;
    case VitalityBucket.Extinct:
      return <WifiOffIcon size="1em" />;
```